### ISSUE TITLE
What do you think about "Simple Schema"?

### ISSUE THREAD

**Issue by thearabbit** [wenzhixin/vue-auto-form#2]
I base on `Meteor JS`, and use [Simple Schema](https://github.com/aldeed/node-simple-schema) to check validation.
What do you think about this?

**Comment by wenzhixin** [comment]
Yes, I used the `Simple Schema` before, but I think it is mainly for mongodb.
Auto form is very simple and convenient, but I can not find a similar library in vue, so I created this repo inspired `meteor-autoform`.

I have used this repo in most of my projects, and I hope it can help others :)

**Comment by thearabbit** [comment]
Look great, This is `SimpleSchema@2 on NPM` not `Meteor`.
So it isn't mainly for mongodb.
I think that it is good if we can use `External Schema` such as `SimpleSchema on NPM, GraphQL...`.
You can check [Uniform](https://github.com/vazco/uniforms)

**Comment by mingliang7** [comment]
Could you implement this component with vue meteor. I use this vue package [akryum:vue](https://atmospherejs.com/akryum/vue). Actutally i have tried your package but it doesn't work in meteor. look at this [issues140](https://github.com/Akryum/vue-meteor/issues/140). it is great if you can combine this into meteor.

**Comment by raphaelarias** [comment]
Meteor and SimpleSchema would be great.
